config/default/kustomization.yaml

```diff
@@ -9,8 +9,8 @@ namespace: capa-system
 namePrefix: capa-
 
 # Labels to add to all resources and selectors.
-#commonLabels:
-#  someName: someValue
+commonLabels:
+  cluster.x-k8s.io/provider: "aws"
```

Comment:
either we should standardize to something like `infrastructure-aws` or `aws-infrastructure`

cc @fabriziopandini

Reply:
Fine with whatever we pick as long as we are consistent here as well: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/725

Reply:
I prefer <type>-<name>, it reads better, and it has a meaning

Reply:
One other potential solution we may have is to have two labels, one for a provider name, and one for a provider type. That said, I'm not overly sold that we need that.

Reply:
I prefer a single label/two labels with the same name across all the providers because this makes the discovery simple, but I will adapt to whatever is decided

Reply:
Ok...Let's do a single label. 
`cluster.x-k8s.io/provider: infrastructure-aws`.

This is consistent formatting with @vincepri suggestions. That is, `controlplane-kubeadm` and `bootstrap-kubeadm` for those components.
The only one that diverges is `cluster-api`. 🤷‍♂